docs/notebooks/multiobjective_constrained.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
```

Comment:
since moo constrained could be in either a moo notebook or a constrained BO notebook, you could split up the notebooks into smaller chunks and then mention other relevant notebooks in each notebook

Reply:
that might also make it easier to browse the notebooks from the website